src/main/sensors/compass.c

```diff
@@ -36,6 +36,7 @@
 #include "drivers/compass/compass.h"
 #include "drivers/compass/compass_ak8975.h"
 #include "drivers/compass/compass_ak8963.h"
+#include "drivers/compass/compass_mpu925x_ak8963.h"
```

Comment:
We try to keep the includes in alphabetical order and/or fix any that are out of order when making changes. Can you please correct the order for `compass_lis3mdl.h` and put this after it so they'll all be in order?